Studio-Lovelies/GG-JointJustice-Unity
# bugfix/Windows x64 build starts in windowed mode, switches to full-screen when entering settings

## Summary

Closes #264 

## Testing instructions

Set game to fullscreen. Quit. Reopen game. Open settings menu. Fullscreen toggle should be ticked.

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource:
